[4.6.0] Add missing information to splunk integration (#6510)
wazuh/wazuh-documentation: source/integrations-guide/splunk/index.rst

   .. note::
      

>       For testing purposes, you can avoid SSL verification by removing the line ``cacert => "/PATH/TO/LOCAL/SPLUNK/ca.pem"`` and adding ``ssl_verification_mode => "none"``.

Reviewer: ```suggestion
      For testing purposes, you can avoid SSL verification by replacing the line ``cacert => "/PATH/TO/LOCAL/SPLUNK/ca.pem"`` with ``ssl_verification_mode => "none"``.
```

   .. note::
      

>       For testing purposes, you can avoid SSL verification by removing the line ``cacert => "</PATH/TO/LOCAL/SPLUNK/CERTIFICATE>/ca.pem"`` and adding ``ssl_verification_mode => "none"``.

Reviewer: ```suggestion
      For testing purposes, you can avoid SSL verification by replacing the line ``cacert => "</PATH/TO/LOCAL/SPLUNK/CERTIFICATE>/ca.pem"`` with ``ssl_verification_mode => "none"``.
```
